src/main/java/lotto/domain/LottoDrawer.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import lotto.view.Prize;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoDrawer {
+    private LottoDrawer() {
+    }
+
+    public static double calProfit(Money price, List<Integer> hitNumber) {
+        BigDecimal totalPrize = BigDecimal.ZERO;
+        for (Integer i : hitNumber) {
+            totalPrize = totalPrize.add(BigDecimal.valueOf(Prize.calculatePrize(i)));
+        }
+        BigDecimal decimalPrice = BigDecimal.valueOf(price.getMoney());
+        BigDecimal profit = BigDecimal.ONE.add(totalPrize.subtract(decimalPrice).divide(decimalPrice, 2, BigDecimal.ROUND_FLOOR));
+        return profit.doubleValue();
```

Comment:
```suggestion
        return price
                .profit(totalPrize(hitNumbers))
                .doubleValue();
```
- 값을 꺼내서 처리하기 보다 객체에 메시지를 보내보는건 어떨까요

Reply:
넵 ! 좋은 방법 알려주셔서 감사합니다 ㅎㅎ